lib/_utils/helpers/validation.dart

```diff
@@ -10,4 +10,24 @@ class Validation {
     }
     return null;
   }
+
+  static String? nameValidation(String? text) {
+    String name = text ?? '';
+
+    if (name.trim().isEmpty) {
```

Comment:
Please put proper validation as I said in the call name must be less than 2 character and not greater than 50 characters.